plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/explorer/OpenShiftExplorerContentProvider.java

```diff
@@ -44,197 +41,157 @@
  * 
  * @author jeff.cantrill
  */
-public class OpenShiftExplorerContentProvider extends BaseExplorerContentProvider 
-	implements PropertyChangeListener, IProjectCache.IProjectCacheListener {
-	
+public class OpenShiftExplorerContentProvider implements ITreeContentProvider {
```

Comment:
ah, here's one of the main design changes: v3 content provider does not extend the base content provider any longer.